pyinfra_cli/inventory.py

```diff
@@ -258,7 +270,7 @@ def make_inventory_from_files(
         for hosts in groups.values():
             # Groups can be a list of hosts or tuple of (hosts, data)
             hosts = _get_any_tuple_first(hosts)
-
+            # TODO: Generator expressions are silently discarded (?)
```

Comment:
I'll take a look at this.

Reply:
Generator expressions do not work (same behavior as on `3.x`):

![image](https://github.com/user-attachments/assets/1d17a4f6-cecb-4840-8971-0096d09f4544)